src/code/z_message.c

```diff
@@ -2562,18 +2561,47 @@ void Message_OpenText(PlayState* play, u16 textId) {
         if (gSaveContext.language == LANGUAGE_ENG) {
             Message_FindMessagePAL(play, textId);
             msgCtx->msgLength = font->msgLength;
+#if PLATFORM_N64
+            if ((B_80121220 != NULL) && (B_80121220->unk_64 != NULL) && B_80121220->unk_64(&play->msgCtx.font)) {
+
+            } else {
+                DmaMgr_RequestSync(font->msgBuf, (uintptr_t)_nes_message_data_staticSegmentRomStart + font->msgOffset,
```

Comment:
I think this ship has sailed long ago but should we be using `DMA_REQUEST_SYNC` for these too, even though we don't have a debug ROM to match that includes them?

Reply:
I think it may make sense purely from a "if I was debugging this n64 only code and I had a DEBUG_FEATURES define turned on" scenario,  id want to see the debug info. But I guess we may have to talk about this separately.

Reply:
I agree we should use `DMA_REQUEST_SYNC` but didn't here because of this feedback: https://github.com/zeldaret/oot/pull/2140#discussion_r1745187622

Reply:
I either didnt see that or didnt consider it fully at the time. Think we should discuss it again. Will merge shortly tho and we can do that separately